CHANGES

```diff
@@ -21,6 +21,7 @@ Changes
  - Add configuration `DOIT_CONFIG` `action_string_formatting` to control command action formatter.
  - Fix `result_dep`, use result **after** its execution
  - Fix #286: Cannot use functools.partial as title
+ - Fix #285: Unexpected targets' cleanup order
```

Comment:
CHANGES entries should read like "what was fixed", not like "issue description".
Same could be said about git commit

- Fix #286: Cannot use functools.partial as title
+ Fix #286: Support functools.partial on task's dict `metadata task.title`
- Fix #285: Unexpected targets' cleanup order
+ Fix #285: clean command, remove files before attempting to remove folders

Reply:
Understood. I had the impression that the other changes quoted the issue's title, but I didn't actually go and check.

I changed the phrasing of #285 solution, let me know what you think.